echopype/echodata/echodata.py

```diff
@@ -193,7 +193,11 @@ def nc_path(self):
             DeprecationWarning,
             2,
         )
-        return self.converted_raw_path
+        if self.converted_raw_path.endswith('.nc'):
+            return self.converted_raw_path
+        else:
+            path = Path(self.converted_raw_path)
+            return str(path.parent / (path.stem + '.nc'))
```

Comment:
What does this do? Isn't `converted_raw_path` either ends with `.nc` or `.zarr`?

Reply:
When converting to nc then to  zarr, we lose the .nc filepath so nc_path would have a .zarr extension. Also, having access to both names was supported in the old version.

Reply:
Great! Thanks for those explanation :)